gcc/rust/expand/rust-expand-visitor.cc

```diff
@@ -18,6 +18,7 @@
 
 #include "rust-expand-visitor.h"
 #include "rust-ast-fragment.h"
+#include "rust-diagnostics.h"
```

Comment:
There's no modification within rust-expand-visitor.cc, is this include really required ?